rear_rider_device/rear_rider_bluetooth_server/src/services/hello_world.py

```diff
@@ -80,3 +104,49 @@ def _increment_counter(self):
         self.PropertiesChanged(GATT_CHRC_IFACE, { 'Value': value }, [])
         self.counter += 1
         return self.notifying
+
+class ConfigCharacteristic(Characteristic):
+    """
+    Configure the LED lights.
+    """
+    TEST_CHRC_UUID = '3bd0b2f7-72f8-4497-bbe5-6bc3db447b95'
+
+    def __init__(self, bus, index, service):
+        Characteristic.__init__(
+                self, bus, index,
+                self.TEST_CHRC_UUID,
+                ['write'],
+                service)
+        self.value = []
+
+    def WriteValue(self, value, options):
+        print('ConfigCharacteristic Write: ' + repr(value))
+        self.value = value
+
+class WifiCharacteristic(Characteristic):
+    """
+    Control Wi-Fi on Pi.
+    """
+    TEST_CHRC_UUID = '3bd0b2f7-72f8-4497-bbe5-6bc3db450b95'
+
+    def __init__(self, bus, index, service):
+        Characteristic.__init__(
+                self, bus, index,
+                self.TEST_CHRC_UUID,
+                ['read', 'write'],
+                service)
+
+    def ReadValue(self, options):
+        wifi = WifiAccessPoint()
+        if wifi.is_on():
+            return dbus.ByteArray('1'.encode('utf8'))
+        else:
+            return dbus.ByteArray('0'.encode('utf8'))
+    
+    def WriteValue(self, value, options):
+        wifi = WifiAccessPoint()
+        state = int(value[0])
```

Comment:
`int(value[0])` does not do as expected. It's weird since I also expected it to parse correctly. I patched it to do what was intended though in a recent commit.